unpause/unpause_test.go

```diff
@@ -100,8 +107,8 @@ func TestUnpauseJWT(t *testing.T) {
 				clk:         fc,
 			},
 			want:               JWTClaims{},
-			wantGenerateJWTErr: true,
-			wantRedeemJWTErr:   false,
+			wantGenerateJWTErr: false,
```

Comment:
Generate takes a Signer now, it's impossible to initialize a Signer with a key that's too small. But, since Redeem still takes a key, we can test that the key is too small so I left the test in. Arguably this shouldn't happen either because the SFE (Redeemer) and the WFE (Generator) should be configured with the same key, so I could remove this whole test if you like.

Reply:
Ah that makes sense! Then I'd say just leave a comment to that effect because by the time I scrolled down to this test input I'd already forgotten that we'd already created the signer.